website/src/content/blog/roadmap-2024.md

```diff
@@ -0,0 +1,486 @@
+---
+title: Roadmap 2024
+description: Roadmap 2024, v1.5.0 and new logo
+summary: Roadmap 2024, v1.5.0 and new logo
+authors:
+  - ema
+  - team
+pubDate: 2024-01-06
+coverImage:
+  lightSrc: ../../assets/blog/roadmap-2024/banner-light.png
+  darkSrc: ../../assets/blog/roadmap-2024/banner-dark.png
+  alt: The Prettier challenge banner, with the Biome logo over it
+socialImage: ../../assets/old-social-logo.png
+---
+## Roadmap 2024
+
+We are thrilled to share what the Core Contributors and Maintainers would like to focus in 2024.
+
+We want to remind that Biome is a community-driven project, so we can't promise that all the ideas outlined below are going to be shipped.
+
+However, if you're excited about some aspects of the project, and you want to see some of them developed faster than others, you can help us in many ways:
+- [**Be involved in the project and the community**](https://github.com/biomejs). Help us to build those features.
```

Comment:
```suggestion
- [**Be involved in the project and the community**](https://github.com/biomejs). Help us build those features.
```

Reply:
"Help us to build" is correct

Reply:
> "Help us to build" is correct

while correct, the version without "to" is more commonly used with "help".